app/main.py

```diff
@@ -1,8 +1,42 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(self, comfort_class: int, clean_mark: int, brand: str):
+        self.comfort_class = comfort_class
+        self.clean_mark = clean_mark
+        self.brand = brand
 
 
 class CarWashStation:
-    # write your code here
-    pass
+    def __init__(self, distance_from_city_center: float, clean_power: int, average_rating: float, count_of_ratings: int):
+        self.distance_from_city_center = distance_from_city_center
+        self.clean_power = clean_power
+        self.average_rating = round(average_rating, 1)
+        self.count_of_ratings = count_of_ratings
+
+
+    def calculate_washing_price(self, car: Car) -> float:
+        calculate = (car.comfort_class * (self.clean_power - car.clean_mark) * self.average_rating)
+        if self.distance_from_city_center:
+            return round(calculate / self.distance_from_city_center, 1)
```

Comment:
The `calculate_washing_price` method should handle the case where `distance_from_city_center` is zero to avoid division by zero errors. Consider adding a check to ensure this value is not zero before performing the division.